docs/editor.md

```diff
@@ -0,0 +1,112 @@
+- `hbenl.vscode-test-explorer` - Let's you navigate our tests, and run them through button presses.
+- `hbenl.vscode-mocha-test-adapter` - The mocha specific module to the common extension mentioned above.
+- `github.vscode-pull-request-github` - With this you can manage and make pull requests right from VSCode, even reviews can be done via the editor.
+- `eamodio.gitlens` - Gives spectacular insight into git history, has many helpful git navigation UI features.
+- `mongodb.mongodb-vscode` - Our VScode extension can be connected to your locally running MongoDB instance to help debug tests etc.
+
+```jsonc
+{
+    "folders": [
+        {
+            "name": "node-driver",
+            "path": "PATH_TO_DRIVER"
+        }
+    ],
+    "settings": {
+        // Automatic Formatting settings
+        "editor.codeActionsOnSave": {
+            "source.fixAll.eslint": true
+        },
+        "[json]": {
+            "editor.formatOnSave": true
+        },
+        "[jsonc]": {
+            "editor.formatOnSave": true
+        },
+        "[javascript]": {
+            "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+        },
+        "[typescript]": {
+            "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+            "editor.codeActionsOnSave": {
+                "source.organizeImports": false
+            }
+        },
+        "files.autoSave": "onFocusChange",
+        "files.trimTrailingWhitespace": true,
+        "files.trimFinalNewlines": true,
+        "files.insertFinalNewline": true,
+        // Testing settings
+        "mochaExplorer.files": "test/unit/**/*.test.js",
+        "mochaExplorer.ui": "test/tools/runner/metadata_ui.js",
+        "mochaExplorer.envPath": null, // Useful for more advanced tests
+        // Typescript settings
+        "typescript.disableAutomaticTypeAcquisition": true,
+        "typescript.tsdk": "./node_modules/typescript/lib",
+        // Editor nice to haves
+        "editor.rulers": [
+            100
+        ],
+        "editor.renderWhitespace": "selection"
+    },
+    "extensions": {
+        "recommendations": [
+            "streetsidesoftware.code-spell-checker",
+            "dbaeumer.vscode-eslint",
+            "hbenl.vscode-test-explorer",
+            "hbenl.vscode-mocha-test-adapter",
+            "github.vscode-pull-request-github",
+            "mongodb.mongodb-vscode",
+            "eamodio.gitlens"
+        ],
+        "unwantedRecommendations": [
+            "esbenp.prettier-vscode"
+        ]
+    }
+}
+```
+
+## Native Extensions
+
+I've highlighted the two non-default things with `CHANGE THIS` comments
+
+You need to add `node_modules/node-addon-api` to the include path to find `napi.h`. (For libmongocrypt the path might be: `bindings/node/node_modules/node-addon-api` depending on your workspace root)
+
+And, lastly, bump up the cpp standard to `c++14`.
```

Comment:
This is also marked in the actual settings - do we need this line? Or should this say we need to bump it to whatever the relevant standard is?

Reply:
Also - are both of the above lines intended to be sub comments on the "non-default" changes? If so, maybe we can indent them with bullets below the top sentence?